storage.go

```diff
@@ -106,7 +106,7 @@ func initDatabaseConnection(configuration StorageConfiguration) (*sql.DB, error)
 // multiple users have disabled some rules.
 func displayMultipleRuleDisable(connection *sql.DB, output string) error {
 	var fout *os.File = nil
-	var writer *bufio.Writer = nil
+	writer := bufio.NewWriter(os.Stdout)
```

Comment:
it does not seem to be correct - it will output to stdout which is not always needed. + the defer might not work as well as expected.

Reply:
done